porcupine/plugins/pluginmanager.py

```diff
@@ -190,7 +190,18 @@ def _on_select(self, junk: object = None) -> None:
             if info.status == pluginloader.Status.IMPORT_FAILED:
                 text = f"Importing the plugin failed.\n\n{info.error}"
             elif info.status == pluginloader.Status.SETUP_FAILED:
-                text = f"The plugin's setup() function failed.\n\n{info.error}"
+                text = "Calling the plugin's setup() function failed.\n\n"
+                if info.error is None:
+                    # no setup(). probably a beginner tries to make a plugin
+                    text += (
+                        "There is no setup() function. Make sure to include a setup function into"
+                        " your plugin.\n\nTo learn more about Porcupine's plugin API, visit"
+                        " https://akuli.github.io/porcupine"
```

Comment:
Why not just set this text to `info.error`?

It might be simplest to just put all this text to `log.error()` in `pluginloader.py`, so that the same exact text shows up in the terminal and in the plugin manager.